packages/venia-concept/src/components/Navigation/navigation.js

```diff
@@ -207,7 +211,7 @@ class Navigation extends PureComponent {
         const { isCreateAccountOpen, isSignInOpen, rootNodeId } = state;
         const { classes, closeDrawer, isOpen, rootCategoryId } = props;
         const className = isOpen ? classes.root_open : classes.root;
-        const isTopLevel = !rootNodeId || rootNodeId === rootCategoryId;
+        const isTopLevel = !rootNodeId || rootNodeId == rootCategoryId;
```

Comment:
Generally we prefer strict equality. Does this address a particular requirement?

Reply:
The ids being returned from graphql were strings. I've switched back to strict equality here (in latest changes), but I'm using `parseInt()` to set the `rootNodeId` now. Will that work, or should I be using a different method for conversion?